content/en/docs/concepts/security/hardening-guide/authorization-hardening.md

```diff
@@ -0,0 +1,89 @@
+  Kubernetes admission control.
+
+- **_Kubernetes Dashboard_** is a web-based UI to interact with applications running in the cluster. Attackers with elevated access to the
+  dashboard can open a shell connection to any pod, view Secrets and can also use the existing resources for cryptojacking attacks. In the
+  cluster role, users should only be provided with the read-only permission to the dashboard and write access should be minimized at all costs.
+
+- **_Kubernetes Secret_** is an object that contains sensitive information, such as a password, token, or key, which is used in the Pod
+  configuration or container image. Unrestricted access of Secrets to the principals which don't require them expands the attack surface. We
+  can use admission controllers to define policies to restrict the access of Secrets only to the required components.
+
+- **_Kubernetes API_** is an HTTP API used to query and modify the status of Kubernetes objects. It also facilitates communication between
+  users, various components inside the cluster and the external parties. Unrestricted accesses to this API may lead to resource modifications, data
+  breaches or even a cluster takeover. RBAC policies with minimum verbs should be assigned to any user.
+
+### Admission Controllers
+
+We can extend the built-in RBAC policies using the validation admission webhooks to strengthen the authorization design. There
+can be situations when we require more policy features or granularity which RBAC or network policies provide. Kubernetes admission controllers
+can help in such a scenario.
+
+A Kubernetes admission controller is a component of code that analyzes requests made to the Kubernetes API server and decides whether to
+approve them or deny them. The request gets evaluated after it has been verified and authorized by the API server before it is granted and
+implemented. This is an optional feature that may only be required for  large-scale clusters or where complex security is required. They
+can be adjustable for many different user-specific scenarios and environments. There are many open source and commercial implementations from
+which organizations can choose and enforce their specific restrictions.
+
+
+### Auditing RBAC Policies
+
+As the application grows, the scope of the components expands, and the required permissions for certain resources also change. There is a very
+There is a high chance that any admin or developer might unknowingly give extra permission to any of the defined roles at any time. Regular reviews of the
+existing RBAC policies are required to check for and prevent such mistakes. We can use the below tools to audit and make a check for risky permissions
+in the existing RBAC policies defined in the cluster.
+
+- [K8s-rbac-audit](https://github.com/cyberark/kubernetes-rbac-audit)
+- [Krane](https://github.com/appvia/krane)
+- [Kubiscan](https://github.com/cyberark/KubiScan)
```

Comment:
If we suggest tools, it may seem that the project is picking "the best" audit tools.
I think we can leave the links to the tools out.
```suggestion
existing RBAC policies are required to check for and prevent such mistakes. 
```
According to the [style guide](https://kubernetes.io/docs/contribute/style/content-guide/#third-party-content):

> Docs can link to third-party open source software (OSS) outside the Kubernetes project only if it's necessary for Kubernetes to function

or 

```suggestion
existing RBAC policies are required to check for and prevent such mistakes. There are open source tools available to audit the cluster's RBAC policies.
```